src/nsp/README.md

```diff
@@ -0,0 +1,5 @@
+# Azure CLI Nsp Extension #
+This is an extension to Azure CLI to manage Nsp resources.
+
+## How to use ##
```

Comment:
Please complete this section.

Reply:
Hi @necusjz , based on the command I used locally, "azdev extension add nsp", I've added "az extension add --name nsp". Can you please confirm where the name 'nsp' will come from and if this is correct?

Reply:
It may come from your custom name of the folder. As it stands for "Network Security Perimeter", make sense.